### PR TITLE
Add new leak regular rule: WPConfig file exposed

### PR DESCRIPTION
**Pull request by fguisso**
**- What I did**
Add a new leak regular rule to find wp-config expressions.

**- How to verify it**
There is a new example file and a new unit test, run: ` go test -v ./development-kit/pkg/engines/leaks/analysis -timeout=2m -parallel=1 -failfast -short`

### CODE REVIEW

**Comment by fguisso** [comment]
I need some help here to understand where Vuln ID is made and a good description for this case, please

File: development-kit/pkg/enums/engine/advisories/leaks/regular/regular.go [hunk 1]
```diff
@@ -461,3 +461,19 @@ func NewLeaksRegularPasswordExposedInHardcodedURL() text.TextRule {
 		},
 	}
 }
+
+func NewLeaksRegularWPConfig() text.TextRule {
+	return text.TextRule{
+		Metadata: engine.Metadata{
+			ID:          "abc",
```
Comment (wiliansilvazup) [inline development-kit/pkg/enums/engine/advisories/leaks/regular/regular.go:468]: Please add a uuid v4 in ID Field how to `5332204a-0d3d-4fe3-a73b-29525101afa0`

**Review by wiliansilvazup — APPROVED** [review]
Tnks for your contribuiting, You rock !

File: development-kit/pkg/enums/engine/advisories/leaks/regular/regular.go (hunk 1 above)
Comment (wiliansilvazup) [reply development-kit/pkg/enums/engine/advisories/leaks/regular/regular.go:468]: This ID field represents a unique identifying key for the vulnerability.

Comment (fguisso) [reply development-kit/pkg/enums/engine/advisories/leaks/regular/regular.go:468]: Okay, but how do you generate this ID? probably I'll add more rules and know the process made things easy, then I dont net do ask for ID every time.

Comment (wiliansilvazup) [reply development-kit/pkg/enums/engine/advisories/leaks/regular/regular.go]: Yeah, You can generate an random uuid v4 of your choice, we get always in https://www.uuidgenerator.net/

**Comment by fguisso** [comment]
Hey guys, I have any chances to receive some swags from the project? https://insights.zup.com.br/hacktoberfest

**Comment by nathannascimentozup** [comment]
> Hey guys, I have any chances to receive some swags from the project? https://insights.zup.com.br/hacktoberfest

Hi @fguisso, You can request the t-shirt through this form (https://zup1.typeform.com/to/zrOF5cpp), for sure your contributions will be considered. Thanks again for the contribution.

p.s: you have to inform a Brazilian address to be able to receive the t-shirt

**Pull request merged**